src/main/java/io/swagger/api/watchlist/WatchlistApi.java

```diff
@@ -44,7 +47,7 @@ public interface WatchlistApi {
     @RequestMapping(value = "/watchlist/{watchlist_id}",
         produces = { "application/json", "application/xml" }, 
         method = RequestMethod.GET)
-    ResponseEntity<Watchlist> watchlistWatchlistIdGet(@Parameter(in = ParameterIn.PATH, description = "The id of the watchlist to retrieve", required=true, schema=@Schema()) @PathVariable("watchlist_id") Integer watchlistId);
+    ResponseEntity<Watchlist> watchlistWatchlistIdGet(@Parameter(in = ParameterIn.PATH, description = "The id of the watchlist to retrieve", required=true, schema=@Schema()) @PathVariable("watchlist_id") UUID watchlistId);
```

Comment:
I grabbed the JSON file from my swagger service, but it doesn't have the same whitespacing as our running `combined-MoviePlaylist-1.0.0-resolved.json` file. If you want to update the file, go for it. Otherwise, I'm okay with this json file getting out of date now that we've generated our source code.